config/manifests/metallb-frr.yaml

```diff
@@ -1927,6 +1927,12 @@ spec:
         - name: TINI_SUBREAPER
           value: "true"
         image: frrouting/frr:v7.5.1
+        livenessProbe:
+          exec:
+            command:
+            - /etc/frr_liveness/liveness.sh
+          initialDelaySeconds: 5
```

Comment:
this seems a bit short ( as a guess ), is there a reason you went with 5 seconds?
also a startupprobe might be more fitting, wdyt?

Reply:
yeah, it makes sense. Will change

Reply:
done